src/modals/view-issue-modal.tsx

```diff
@@ -116,7 +130,10 @@ export const ViewIssueModal = ({
           )}
         </Box>
       </Box>
-      <PaddedText text=" Open issue in browser: o | Close: q" maxLength={144} />
+      <PaddedText
+        text=" Open issue in browser: o | Update assignee: a | Update priority: p | Close: q"
```

Comment:
Nice to have (other ticket maybe): component to convert keybindings into a string like this